packages/realm/src/App.ts

```diff
@@ -18,20 +18,56 @@
 
 import { binding, fs } from "./internal";
 
+/**
+ * This describes the options used to create a Realm.App instance.
+ * @property id - The id of the Atlas App Services application.
+ * @property baseUrl - The base URL of the Atlas App Services server.
+ * @property timeout - General timeout (in millisecs) for requests. TODO
```

Comment:
Not sure if these fields are going to be added in the future.

Reply:
Most likely.